schema-engine/connectors/sql-schema-connector/src/flavour/postgres.rs

```diff
@@ -388,27 +367,8 @@ impl SqlFlavour for PostgresFlavour {
         self.raw_cmd(sql)
     }
 
-    fn drop_database(&mut self) -> BoxFuture<'_, ConnectorResult<()>> {
-        Box::pin(async move {
-            let params = self.state.get_unwrapped_params();
-            let mut url =
-                Url::parse(&params.connector_params.connection_string).map_err(ConnectorError::url_parse_error)?;
-            let db_name = url.path().trim_start_matches('/').to_owned();
-            assert!(!db_name.is_empty(), "Database name should not be empty.");
-
-            strip_schema_param_from_url(&mut url);
-            let (mut admin_conn, admin_url) = create_postgres_admin_conn(url.clone()).await?;
-
-            admin_conn
-                .raw_cmd(&format!("DROP DATABASE \"{db_name}\""), &admin_url)
-                .await?;
-
-            Ok(())
-        })
-    }
-
     fn drop_migrations_table(&mut self) -> BoxFuture<'_, ConnectorResult<()>> {
-        Box::pin(self.raw_cmd("DROP TABLE _prisma_migrations"))
+        self.raw_cmd("DROP TABLE _prisma_migrations")
```

Comment:
As discussed in a call, we were previously applying `Box::pin` twice by accident, so this change is fine.